.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
`.idea` directory is PyCharm specific and should not be stored in git repository. Use `.gitignore` to ingore it.